js/_website/src/routes/redirects.js

```diff
@@ -1,5 +1,7 @@
 export const redirects = {
-	"/guides/creating-a-new-component": "/guides/five-minute-guide",
+	"/guides/creating-a-new-component":
+		"/guides/custom-components-in-five-minutes",
+	"/guides/five-minute-guide": "/guides/custom-components-in-five-minutes",
```

Comment:
this (and the redirect before it) are breaking the website currently because they try to redirect to "/guides/custom-components-in-five-minutes" which doesn't exist yet (it's still /main/guides/custom-components-in-five-minutes until the guide is included in a released version) I would just remove them until we release or redirect to main.

Reply:
Thanks @aliabd !